src/main/scala/beam/utils/logging/ExponentialLoggerWrapperImpl.scala

```diff
@@ -13,37 +13,37 @@ class ExponentialLoggerWrapperImpl(name: String) extends LoggerWrapper {
 
   override def error(msgTemplate: String, args: Any*): Unit = {
     if (logger.isErrorEnabled) {
-      processMessage(msgTemplate, args) { message =>
+      processMessage(msgTemplate) { message =>
         logger.error(message, args.map(_.asInstanceOf[AnyRef]): _*)
       }
     }
   }
 
   def warn(msgTemplate: String, args: Any*): Unit = {
     if (logger.isWarnEnabled) {
-      processMessage(msgTemplate, args) { message =>
+      processMessage(msgTemplate) { message =>
         logger.warn(message, args.map(_.asInstanceOf[AnyRef]): _*)
       }
     }
   }
 
   override def info(msgTemplate: String, args: Any*): Unit = {
     if (logger.isInfoEnabled) {
-      processMessage(msgTemplate, args) { message =>
+      processMessage(msgTemplate) { message =>
         logger.info(message, args.map(_.asInstanceOf[AnyRef]): _*)
       }
     }
   }
 
   override def debug(msgTemplate: String, args: Any*): Unit = {
     if (logger.isDebugEnabled) {
-      processMessage(msgTemplate, args) { message =>
+      processMessage(msgTemplate) { message =>
         logger.debug(message, args.map(_.asInstanceOf[AnyRef]): _*)
       }
     }
   }
 
-  private def processMessage(messageTemplate: String, args: Any*)(func: Func): Unit = {
+  private def processMessage(messageTemplate: String)(func: Func): Unit = {
```

Comment:
This one could be formatted before reviewing, however I believe that since this is primarily being used to figure uniqueness then we want to go off of template. as it is today.